pkg/config/config_test.go

```diff
@@ -20,10 +20,12 @@ func TestGetImagesFromJSONFile(t *testing.T) {
 		name: "Unmarshal images from file",
 		path: "images_file",
 		imagesContent: `{
-    "cloudControllerManagerAWS": "registry.ci.openshift.org/openshift:aws-cloud-controller-manager",
-    "cloudControllerManagerOpenStack": "registry.ci.openshift.org/openshift:openstack-cloud-controller-manager"
-}`,
+			"cloudControllerManagerOperator": "quay.io/openshift/origin-cluster-cloud-controller-manager-operator",
+    		"cloudControllerManagerAWS": "registry.ci.openshift.org/openshift:aws-cloud-controller-manager",
```

Comment:
Should we make sure the pattern matches the others? Use the registry.ci tag instead?